src/main/java/io/confluent/connect/jdbc/source/JdbcSourceTask.java

```diff
@@ -216,6 +235,24 @@ public void start(Map<String, String> properties) {
     running.set(true);
   }
 
+  private void partitionsForTable(List<Map<String, String>> partitions, String table) {
+    TableId tableId = dialect.parseTableIdentifier(table);
+    Map<String, String> partition = Collections.singletonMap(
+        JdbcSourceConnectorConstants.TABLE_NAME_KEY,
+        tableId.tableName()
+    );
+    partitions.add(partition);
+
+    String fqn = ExpressionBuilder.create().append(tableId, false).toString();
+    Map<String, String> partitionWithFqn = new HashMap<>();
+    partitionWithFqn.put(JdbcSourceConnectorConstants.TABLE_NAME_KEY, fqn);
+    partitionWithFqn.put(
+        JdbcSourceConnectorConstants.OFFSET_PROTOCOL_VERSION,
+        JdbcSourceConnectorConstants.PROTOCOL_VERSION_ONE)
+    ;
+    partitions.add(partitionWithFqn);
```

Comment:
Why are we adding both the single table name and FQN as separate partitions in the map?